### ISSUE TITLE
Ivus/Lok'holar - Should be able to loot if NPCs kill them

### ISSUE THREAD

**Issue by Sabre142** [elysium-project/server#2104]
I'm not sure if this is solid proof:

>I collected 5 of these rings by pulling Ivus to the horde leader and letting the NPCs kill him.

http://web.archive.org/web/20060216104350/http://wow.allakhazam.com:80/db/item.html?witem=19109
He brags about how easily he died so I would say it's a heavily NPC based kill.

Right now you can only loot the bosses in AV if players get 50% of damage in.

As a sidenote the drops are meant to be rolls too.

**Comment by Heyhohohoho** [comment]
Agreed, this unlootable issue also affects bosses in the ZF temple event, if the npcs do too much damage you cannot loot them.

As for AV, as long as your faction tagged him he should be lootable imo.
